Update docs 1.0.0 beta (#825)
repository-service-tuf/repository-service-tuf: docs/source/guide/deployment/setup.rst

.. collapse:: See settings details

>     * Timestamp, Snapshot, and  Targets metadata expiration policy

Reviewer: ```suggestion
    * Timestamp, Snapshot, and Targets metadata expiration policy
```

>         Defines how long this metadata is valid. The metadata is invalid when it

Reviewer: ```suggestion
        Defines how many days this metadata is valid. The metadata is invalid when it
```

        expires.

    * Delegations

        - Bins (online key only)

          The target metadata file might contain a large number of artifacts.
          The target role delegates trust to the hash bin roles to
          reduce the metadata overhead for clients.

          This metadata is signed using the online key.

        - Custom Delegations (online/offline keys)

          Allows the RSTUF admin to create custom delegation that can use the
          online key or offline key(s) to sign the metadata.
          The custom delegation can be used to define the roles and paths for
          the target metadata.

    * Root metadata expiration policy

        Defines how long this metadata is valid, for example, 365 days (year).
        This metadata is invalid when it expires.

    * Root threshold

>         It defines the number of keys required to sign the Root metadata.

Reviewer: ```suggestion
        It defines the number of keys required to sign the Root metadata
        before it's considered trusted and will be published.
```

>         The minimum number of keys required to update and sign the TUF Root
>         metadata. It's required to be at least 2.

Reviewer: ```suggestion
        That's the minimum number of keys required to update and sign the TUF Root
        metadata. It's required to be at least 2.
```
